docs/04-Bar-Chart.md

```diff
@@ -49,6 +49,7 @@ borderSkipped | `String or Array<String>` | Which edge to skip drawing the borde
 hoverBackgroundColor | `Color or Array<Color>` | Bar background color when hovered
 hoverBorderColor | `Color or Array<Color>` | Bar border color when hovered
 hoverBorderWidth | `Number or Array<Number>` | Border width of bar when hovered
+stack | `String` | For grouping stacked bars
```

Comment:
I mean simply change the documentation: `identifier to group bars in stack(s)` (or similar) but keep the property name simple: `stack`. About the `stack vs group`, `group` would make sense if this concept could be implemented for every types of chart, which I'm not sure it's possible, right?